server/src/test/java/org/apache/calcite/test/ServerTest.java

```diff
@@ -428,7 +428,7 @@ static Connection connect() throws SQLException {
           + " h varchar(3) not null,\n"
           + " i varchar(3),\n"
           + " j int not null as (char_length(h)) virtual,\n"
-          + " k varchar(3) null as (trim(i)) virtual)";
+          + " k varchar(3) null as (rtrim(i)) virtual)";
```

Comment:
Changed from `trim` to `rtrim` because i also want to test operators for sql dialect, here it is the `ORACLE`.